components/event-publisher-proxy/pkg/sender/jetstream.go

```diff
@@ -99,8 +101,16 @@ func (s *JetstreamMessageSender) eventToNatsMsg(event *event.Event) (*nats.Msg,
 	}
 
 	return &nats.Msg{
-		Subject: event.Type(),
+		Subject: s.getJsSubjectToPublish(event.Type()),
 		Header:  header,
 		Data:    eventJSON,
 	}, err
 }
+
+// GetJsSubjectToSubscribe appends stream name to subject if needed.
```

Comment:
nit: should be `getJsSubjectToSubscribe appends stream name to subject if needed.`